.github/workflows/container-image.yml

```diff
@@ -36,10 +64,48 @@ jobs:
       - name: Check out the repo
         uses: actions/checkout@v3
 
-      - name: Build and push
+      - name: Build and push operator
+        id: build-push-operator
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta-operator.outputs.tags }}
+          labels: ${{ steps.meta-operator.outputs.labels }}
+
+      - name: Setup Go ${{ env.GOLANG_VERSION }}
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GOLANG_VERSION }}
+
+      - name: Install the Operator SDK CLI
+        run: |
+          export OPERATOR_SDK_VERSION=$(cat config/hack/OPERATOR_SDK_VERSION)
+          export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+          export OS=$(uname | awk '{print tolower($0)}')
+          export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}
+          curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+          mkdir -p ${HOME}/.local/bin
+          chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} ${HOME}/.local/bin/operator-sdk
+          echo "${HOME}/.local/bin" >> $GITHUB_PATH
+
+      - name: Generate the bundle content
+        run: make bundle
+        env:
+          IMG: ghcr.io/qbarrand/oot-operator@${{ steps.build-push-operator.outputs.digest }}
```

Comment:
I would in fact do that outside this workflow and just compare if that command generated a diff on the bundle files.
The git repo should always be the source of truth.

Reply:
It will generate a diff because the `IMG` variable is linked to the new operator image.
The `bundle` folder is generate from the kustomize files, so the source of truth is the `config` folder.

Reply:
Yes. I've added it to `.gitignore`.